lib/Yam/Installation/Upgrade/SelectUpdatePage.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handle page that allows to select system/partition to be upgraded.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Installation::Upgrade::SelectUpdatePage;
```

Comment:
as at the end we will end up with many pages and we already have in the screen quite nice wording in English for selecting name I would recommend to use literally what we find there for the purpose of potential disambiguation, it helps a lot to orientate in the code when everyone is following, in this case "SelectForUpdatePage".